File: 2017PACNW/my_solutions/L/l.cpp
#include <cstdio>
#include <cmath>
#include <iostream>

using namespace std;

int main() {
	int k,p,x;
	cin >> k >> p >> x;
	double m =  round(sqrt(k*p/(double) x));
	printf("%.3f\n", m*x+k*p/m);
	return 0;
}

File: 2017PACNW/my_solutions/L/L.java
import java.util.*;

public class L {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int k = sc.nextInt(), p =sc.nextInt(), x =sc.nextInt();
		double m = Math.round(Math.sqrt(k*p/(double)x));
		System.out.printf("%.3f\n",m*x+k*p/m);
	}
}

